src/main/java/uk/gov/hmcts/darts/common/entity/TranscriptionEntity.java

```diff
@@ -81,6 +83,13 @@ public class TranscriptionEntity extends CreatedModifiedBaseEntity {
     @OneToMany(cascade = {PERSIST, MERGE}, mappedBy = TranscriptionWorkflowEntity_.TRANSCRIPTION)
     private List<TranscriptionWorkflowEntity> transcriptionWorkflowEntities = new ArrayList<>();
 
+    @OneToOne
+    @JoinTable(name = "transcription_document",
+        joinColumns = {@JoinColumn(name = "tra_id")},
+        inverseJoinColumns = {@JoinColumn(name = "tra_id")})
+    private TranscriptionDocumentEntity transcriptionDocument;
```

Comment:
This doesn't look right to me - I have added the relationship below
`@OneToMany(mappedBy = TranscriptionDocumentEntity_.TRANSCRIPTION)`

Reply:
Changed to use the mapped relationship